<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AUIParameters</class>
 <widget class="QDockWidget" name="AUIParameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>599</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>599</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>524287</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AUI Parameters</string>
  </property>
  <widget class="QWidget" name="AUIParams">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="contents">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::South</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Tab1">
       <attribute name="title">
        <string>Control</string>
       </attribute>
       <layout class="QVBoxLayout" name="tab1Layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QGroupBox" name="episodes">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Episodes</string>
          </property>
          <layout class="QGridLayout" name="episodesLayout">
           <property name="margin">
            <number>1</number>
           </property>
           <item row="1" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="startEpisodeButton">
             <property name="minimumSize">
              <size>
               <width>44</width>
               <height>44</height>
              </size>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="2" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="resetEpisodeButton">
             <property name="minimumSize">
              <size>
               <width>44</width>
               <height>44</height>
              </size>
             </property>
             <property name="autoFillBackground">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>New</string>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
            <widget class="QPushButton" name="stopEpisodeButton">
             <property name="minimumSize">
              <size>
               <width>44</width>
               <height>44</height>
              </size>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="3">
            <widget class="QLCDNumber" name="episodeTimer">
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>44</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="userStatus">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>User Status</string>
          </property>
          <layout class="QVBoxLayout" name="userStatusLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QGroupBox" name="stress">
             <property name="title">
              <string>Stress Level</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="stressLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="margin">
               <number>1</number>
              </property>
              <item>
               <widget class="QProgressBar" name="stressLevel">
                <property name="mouseTracking">
                 <bool>true</bool>
                </property>
                <property name="maximum">
                 <number>10</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="stressSlider">
                <property name="maximum">
                 <number>10</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksAbove</enum>
                </property>
                <property name="tickInterval">
                 <number>1</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="cognitiveLoad">
             <property name="title">
              <string>Cognitive Load</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="cognitiveLoadLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="margin">
               <number>1</number>
              </property>
              <item>
               <widget class="QProgressBar" name="cognitiveLoadLevel">
                <property name="maximum">
                 <number>10</number>
                </property>
                <property name="value">
                 <number>0</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="cognitiveLoadSlider">
                <property name="maximum">
                 <number>10</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksAbove</enum>
                </property>
                <property name="tickInterval">
                 <number>1</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="taskContext">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Task Context</string>
          </property>
          <layout class="QVBoxLayout" name="taskLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <widget class="QComboBox" name="currentContext">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>44</height>
              </size>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QComboBox::AdjustToContents</enum>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>&lt;Select Task&gt;</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Forward navigation</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Backward navigation</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Camera inspection</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Map review</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>New screenshot</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="currentWidget">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>30</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="text">
              <string>&lt;No Widget&gt;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="robotStatus">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string>Robot status</string>
          </property>
          <layout class="QVBoxLayout" name="AUIStatusLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="margin">
            <number>1</number>
           </property>
           <item>
            <widget class="QGroupBox" name="AUIbatteryLevel">
             <property name="title">
              <string>Battery</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="AUIbatteryLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="margin">
               <number>1</number>
              </property>
              <item>
               <widget class="QProgressBar" name="AUIbattery">
                <property name="mouseTracking">
                 <bool>true</bool>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="batterySlider">
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksAbove</enum>
                </property>
                <property name="tickInterval">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="AUIwifiLevel">
             <property name="title">
              <string>WiFi</string>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <layout class="QVBoxLayout" name="AUIwifiLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="margin">
               <number>1</number>
              </property>
              <item>
               <widget class="QProgressBar" name="AUIwifi">
                <property name="mouseTracking">
                 <bool>true</bool>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="textVisible">
                 <bool>true</bool>
                </property>
                <property name="textDirection">
                 <enum>QProgressBar::TopToBottom</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="wifiSlider">
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksAbove</enum>
                </property>
                <property name="tickInterval">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>stressSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>stressLevel</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>157</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>161</x>
     <y>227</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>wifiSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>AUIwifi</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>batterySlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>AUIbattery</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>696</y>
    </hint>
    <hint type="destinationlabel">
     <x>135</x>
     <y>660</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cognitiveLoadSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>cognitiveLoadLevel</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>96</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
